client/components/TestPlanReportStatusDialog/index.jsx

```diff
@@ -77,6 +93,60 @@ const TestPlanReportStatusDialog = ({
             return [matched, unmatchedTestPlan, unmatchedRequired];
         }, [testPlanReports, requiredReports]);
 
+    // section:
+    if (newOthers.length > 0) {
+        let index = 0;
+        newOthers.forEach(otherReport => {
+            if (matchedReports.length > 0) {
+                matchedReports.forEach(matchedReport => {
+                    if (
+                        otherReport.at.id === matchedReport.id &&
+                        otherReport.browser.id === matchedReport.browser.id
+                    ) {
+                        delete newOthers[index];
+                    }
+                });
+            }
+            index += 1;
+        });
+    }
+
+    if (newOthers.length > 0) {
+        let index = 0;
+        newOthers.forEach(otherReport => {
+            if (unmatchedTestPlanReports.length > 0) {
+                unmatchedTestPlanReports.forEach(unmatchedTestPlanReport => {
+                    if (
+                        otherReport.at.id === unmatchedTestPlanReport.at.id &&
+                        otherReport.browser.id ===
+                            unmatchedTestPlanReport.browser.id
+                    ) {
+                        delete newOthers[index];
+                    }
+                });
+            }
+            index += 1;
+        });
+    }
+
+    if (newOthers.length > 0) {
```

Comment:
I noticed that the condition on this line is the same as 114, so I think probably the two blocks can be merged into one.

Reply:
Hey @alflennik I am not seeing a good way to merge the two blocks. Give me an idea of how to approach this.

Reply:
Sure! Seems like you could delete lines 112-114, and remove the "let" from line 115.

Reply:
Ohh ok. I just combined all three blocks.

Reply:
Cool thanks